compose/magento-2/bin/mutagen

```diff
@@ -48,6 +49,8 @@ elif [[ "$1" == "start" ]]; then
     # please note that docker is alpha here,
     # so it will rewrite your unsynced files on flush command in case of any issues with sync from local to docker
     mutagen create \
+        --label=mage2click-sync=vendor \
+        --scan-mode=accelerated \
```

Comment:
@shkoliar we do not need  --scan-mode=accelerated for vendor since the file watch is disabled for this session